docs/fsharp-cheatsheet.md

```diff
@@ -460,7 +587,7 @@ Note that `finally` does not follow `with`. `try..with` and `try..finally` are s
 
 This example is a basic class with (1) local let bindings, (2) properties, (3) methods, and (4) static members.
 
-    type Vector(x : float, y : float) =
+    type Vector(x: float, y: float) =
         let mag = sqrt(x * x + y * y) // (1)
         member _.X = x // (2)
```

Comment:
normally FSharpLint etc should tell you to make this a `val` - perhaps one can be a val and another a property that does something semi-expensive?

Reply:
I'll leave this for now as I plan to come back to classes at some point.